R/labels.R

```diff
@@ -0,0 +1,67 @@
+#' Adding Labels To Variables For Model
+#'
+#' @description `r lifecycle::badge("experimental")`
+#'
+#' @param vars (`list`)\cr variables to use.
+#' @param data (`data.frame`)\cr data to use.
+#' @param x (`character`)\cr an element in vars.
+#' @return list of variables with labels
+#'
+#' @name labels
+NULL
+
+#' @describeIn labels checks if element in vars is not NULL.
```

Comment:
```suggestion
#' @describeIn labels checks if element in `vars` is not `NULL`.
```
similar please below, basically always put code inside backticks in the documentation